packages/moonstone/VirtualList/VirtualList.js

```diff
@@ -40,16 +42,25 @@ const VirtualList = kind({
 		 * <VirtualList itemSize={ri.scale(72)} />
 		 * ```
 		 *
-		 * @type {Number}
+		 * @type {Number|ui/VirtualList.itemSizesShape}
```

Comment:
@webOS101 - do you know off hand if this will render correctly in our docs?

Reply:
I'll test it.

Reply:
It works splendidly.